modules/cfe_testcase/src/sb_sendrecv_test.c

```diff
@@ -262,22 +267,109 @@ void TestZeroCopyTransmitRecv(void)
     CFE_UtAssert_MSGID_EQ(MsgId, CFE_FT_CMD_MSGID);
     UtAssert_ADDRESS_EQ(MsgBuf, CmdBuf); /* should be the same actual buffer (not a copy) */
 
-    UtAssert_INT32_EQ(CFE_SB_ReceiveBuffer(&MsgBuf, PipeId1, 100), CFE_SB_TIME_OUT);
+    UtAssert_INT32_EQ(CFE_SB_ReceiveBuffer(&MsgBuf, PipeId1, CFE_SB_POLL), CFE_SB_NO_MESSAGE);
 
     UtAssert_INT32_EQ(CFE_SB_ReceiveBuffer(&MsgBuf, PipeId2, 100), CFE_SUCCESS);
```

Comment:
Should this use CFE_SB_POLL?

Reply:
Nevermind... need an actual test w/ a timeout so this is good.  I was just confused.